plugins/test/fu-test-plugin.c

```diff
@@ -229,6 +229,21 @@ fu_test_plugin_write_firmware(FuPlugin *plugin,
 		fu_device_sleep(device, 1);
 		fu_progress_set_percentage_full(progress, i, self->delay_decompress_ms);
 	}
+
+	/* send an interactive request, and wait 5 seconds */
+	if (g_strcmp0(test, "request") == 0) {
+		g_autoptr(FwupdRequest) request = fwupd_request_new();
+		fwupd_request_set_kind(request, FWUPD_REQUEST_KIND_IMMEDIATE);
+		fwupd_request_set_id(request, FWUPD_REQUEST_ID_REMOVE_REPLUG);
+		fwupd_request_add_flag(request, FWUPD_REQUEST_FLAG_ALLOW_GENERIC_MESSAGE);
+		fwupd_request_set_message(request,
+					  "Please pretend to remove the device you cannot see or "
+					  "touch and please re-insert it. You have 5 seconds.");
+		if (!fu_device_emit_request(device, request, progress, error))
+			return FALSE;
+		g_usleep(5 * G_USEC_PER_SEC);
+	}
+
```

Comment:
can we add an explicit test into the installed-tests test suite for this?

Reply:
and if adding to installed tests, I suggest decreasing it to "1s"

Reply:
Good idea, although, phew! Done.